scripts/ci/ci_flake8.sh

```diff
@@ -22,16 +22,21 @@ MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 export AIRFLOW_CI_SILENT=${AIRFLOW_CI_SILENT:="true"}
 
-export PYTHON_VERSION=3.5
```

Comment:
I'm not sure about this. If I'm following the code right removing this will autodetect the python version to use based on the host python. But we _explicitly_ want to run these tests agsinst 3.5 locally. If I have python 3.7 locally it would attempt to use that version which isn't what we want.

Reply:
Right. I moved it up to breeze and Travis but indeed forcing it should be done in the scripts.